cli.c:258:9: error: conflicting types for 'cli_getline'
ssize_t cli_getline(char **LINEPTR, size_t *N, FILE *STREAM)
        ^
cli.c:160:12: note: previous implicit declaration is here
      if ( cli_getline( &note, &note_size, stdin) == -1)
           ^
--- src/cli.c.orig	2018-11-05 05:42:00.000000000 -0600
+++ src/cli.c	2018-11-05 07:03:21.000000000 -0600
@@ -29,6 +29,7 @@
 #include "output.h"
 #include "sand.h"
 
+ssize_t cli_getline(char **LINEPTR, size_t *N, FILE *STREAM);
 
 /* cli_output - output of a text in cli mode*/
 /* MDIARY */
